src/utils/data.ts

```diff
@@ -269,7 +269,7 @@ export const fetchByXPath = (options: FetchByXPathOptions): Promise<MxO[]> => ne
         callback: resolve,
         error: error => reject(`An error occurred while retrieving data via XPath (${xpath}): ${error.message}`),
         filter: {
-            sort: sortAttribute && sortAttribute.indexOf("/") === -1 ? [ [ sortAttribute, sortOrder || "asc" ] ] : [],
+            sort: sortAttribute && !window.mx.isOffline() ? [ [ sortAttribute, sortOrder || "asc" ] ] : sortAttribute && sortAttribute.indexOf("/") === -1 ? [ [ sortAttribute, sortOrder || "asc" ] ] : [],
```

Comment:
Isn't this the same (and more readable):
```suggestion
            sort: sortAttribute && (!window.mx.isOffline() || sortAttribute.indexOf("/") === -1) ? [ [ sortAttribute, sortOrder || "asc" ] ] : [],
```